source/about/deprecated-features.rst

```diff
@@ -11,6 +11,11 @@ Mattermost Server v10.3.0
 
 - As part of our continuous efforts to enhance your user experience, we will be phasing out the Classic Mobile App. Please download the new v2 Mobile App from the `Apple App Store <https://apps.apple.com/us/app/mattermost/id1257222717>`_ or `Google Play Store <https://play.google.com/store/apps/details?id=com.mattermost.rn>`_ to continue enjoying our services without interruption. See more details in `this forum post <https://forum.mattermost.com/t/classic-mobile-app-deprecation/18703>`_.
 
+Mattermost Server v10.2.0
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Docker Content Trust (DCT) for signing Docker image artifacts will be replaced by Sigstore Cosign in our upcoming release, v10.2 (November, 2024). If you rely on artifact verification using DCT, please transition to using Cosign.
```

Comment:
@amyblais do we have a link to a resource (maybe from Docker themselves) on how to transition to Cosign?

@cwarnermm as always favour a note-box and link-out on this guidance.

Reply:
@nab-77 I will add a link to the forum post once published https://docs.google.com/document/d/1ro_skgRqtDVzul_ukmkgu23SgLtdBl4EQsPZnQIho5Y/edit. I'll add a link to the docs as well https://edu.chainguard.dev/open-source/sigstore/cosign/how-to-install-cosign/.